.github/CHANGELOG.md

```diff
@@ -126,32 +191,94 @@
   Total shots:  100
   Total shots:  200
   Total shots:  300
+  Total shots:  300
+  ```
+
+<h4>Containerization support</h4>
+
+* Docker support for building PennyLane with support for all interfaces (TensorFlow,
+  Torch, and Jax), as well as device plugins and QChem, for GPUs and CPUs, has been added.
+  [(#1391)](https://github.com/PennyLaneAI/pennylane/pull/1391)
+
+  The build process using Docker and Makefile works as follows:
```

Comment:
```suggestion
  The build process using Docker and `make` requires that the repository source code
  is cloned or downloaded from GitHub, and provides the following options:
```

Reply:
@antalszava I just noticed something; the following instructions are also provided in the README, however they shouldn't be there - we made a concerted effort to simplify the README a couple of releases ago!

I recommend the following:

- Remove the docker section from the README
- Instead, add it to the developer guide as a new card here: https://pennylane.readthedocs.io/en/latest/development/guide.html
- Add a short link in the readme
- Add a short link here in the changelog, and remove the full list of commands

Reply:
Good point! Removed the docker section from the README. It also seems like we could add the description about Docker to the installation parts of the development guide and the user installation section. Let me open a separate PR.